memory/memory_store.py

```
@@ -1,21 +1,273 @@
 # memory_store.py
 import json
 import os
+from dotenv import load_dotenv
+from pymongo import MongoClient
+from datetime import datetime
+from typing import Dict, Any, Optional, List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)